modules/apps/headless/headless-portal-instances/headless-portal-instances-impl/rest-openapi.yaml

```diff
@@ -0,0 +1,204 @@
+components:
+    schemas:
+        PortalInstance:
+            # @review
+            properties:
+                active:
```

Comment:
no id or is it companyId?

Reply:
It's companyId, we should call it portalInstanceId but I wanted to emphasize that the id belongs to the company since this concept is useful for setting the proper companyThreadLocal for example.